crates/biome_analyze/src/diagnostics.rs

```diff
@@ -164,15 +167,33 @@ impl SuppressionDiagnostic {
             range,
             message: message.to_string(),
             tags: DiagnosticTags::empty(),
+            advice: SuppressionAdvice::default(),
         }
     }
 
-    pub(crate) fn with_tags(mut self, tags: DiagnosticTags) -> Self {
-        self.tags |= tags;
+    pub(crate) fn note(mut self, message: impl Into<String>, range: impl Into<TextRange>) -> Self {
```

Comment:
This new API was created to suppose this new diagnostic: https://github.com/biomejs/biome/pull/4306/files#diff-6b7a758a5c6e91dd2722ccc423ad136ff4e811311c0d49b80c659d61cc54e2a6

Neat :)